mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -53,7 +53,8 @@ final class Bucketizer @Since("1.4.0") (@Since("1.4.0") override val uid: String
    * Values at -inf, inf must be explicitly provided to cover all Double values;
    * otherwise, values outside the splits specified will be treated as errors.
    *
-   * See also [[handleInvalid]], which can optionally create an additional bucket for NaN values.
+   * See also [[handleInvalid]], which can optionally create an additional bucket for NaN/NULL
```

Comment:
This sounds like a behavior change, we should add an item in migration guide of ML docs.

Reply:
@viirya done.